integration-tests/utils/integration-utils.js

```diff
@@ -157,7 +157,7 @@ const fillInReplace = async (page, xpath, text) => {
 
 const select = async (page, labelContains, text) => {
   await click(page, input({ labelContains }))
-  return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
+  return click(page, `//div[starts-with(@id, "react-select-") and @role="option" and contains(normalize-space(.),"${text}")]`)
```

Comment:
After removing the override for the ID attribute on the menu element, this selector matched the menu itself instead of the specific option.